### PR TITLE
plugin: hexo-renderer-scss

### PR DESCRIPTION
**Pull request by noraj**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [x] I want to publish someone else plugin on Hexo official website.
    - [x] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [x] `name` is unique.
    - [x] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)

## Description

Adding : 

- scss : [npm](https://www.npmjs.com/package/hexo-renderer-scss), [github](https://github.com/o2team/hexo-renderer-scss)

### CODE REVIEW

File: source/_data/plugins.yml [hunk 1]
```diff
@@ -2419,3 +2419,11 @@
     - delopyer
     - qiniu
     - 七牛云
+- name: hexo-renderer-scss
+  description: "A hexo plugin for node-scss"
+  link: https://github.com/o2team/hexo-renderer-scss
```
Comment (curbengh) [inline source/_data/plugins.yml]: It redirects to https://github.com/mamboer/hexo-renderer-scss.

File: source/_data/plugins.yml [hunk 2]
```diff
@@ -2419,3 +2419,11 @@
     - delopyer
     - qiniu
     - 七牛云
+- name: hexo-renderer-scss
+  description: "A hexo plugin for node-scss"
```
Comment (curbengh) [inline source/_data/plugins.yml]: Quote is not required,

File: source/_data/plugins.yml (hunk 1 above)
Comment (noraj) [reply source/_data/plugins.yml]: Correct

File: source/_data/plugins.yml (hunk 2 above)
Comment (noraj) [reply source/_data/plugins.yml]: I removed them

**Comment by noraj** [comment]
Can we merge this now?

PS : I want to join hexo org.

**Comment by stevenjoezhang** [comment]
Is it necessary to consult the owner of the plugin?

**Pull request merged**